packages/common/src/utils/ConstantsUtil.ts

```diff
@@ -15,5 +15,6 @@ export const ConstantsUtil = {
     eip155: 'Ethereum',
     solana: 'Solana',
     polkadot: 'Polkadot'
-  } as const satisfies Record<ChainNamespace, string>
+  } as const satisfies Record<ChainNamespace, string>,
+  USDT_CONTRACT_ADDRESS: '0xdac17f958d2ee523a2206206994597c13d831ec7'
```

Comment:
Where do we get this address? Should this be required to maintained in the future? What it represents?

Can you add a JSDoc on top of it with a description and a resource (if any)?

Reply:
@enesozturk That's Tether USD contract address, it's needed to make usdt transactions. In SendController, its the `tokenAddress`.

The address is immutable so it doesn't require maintenance but now that i see, each network has a different address, so this will only work for mainnet 🤔 

We'll need to think another solution

Reply:
maybe checking the name & symbol?

Reply:
Weird how it only works on just a few chains. 
Might be worth adding them all

Reply:
I added the ones i've found. Can you guys re-check the list? and see if you can find another one? I couldn't find a good list